cahiers-mondes-anciens.csl

```diff
@@ -0,0 +1,261 @@
+  <macro name="auteurs-secondaires">
+    <choose>
+      <if type="chapter paper-conference" match="none">
+        <group delimiter=". ">
+          <choose>
+            <if variable="author">
+              <names variable="editor" delimiter=". ">
+                <name and="text" sort-separator=" " delimiter=", " initialize-with=". "/>
+                <label form="short" prefix=" (" suffix=")"/>
+              </names>
+            </if>
+          </choose>
+          <choose>
+            <if variable="author editor" match="any">
+              <names variable="translator" delimiter=". ">
+                <name and="text" sort-separator=" " delimiter=", " initialize-with=". "/>
+                <label form="short" prefix=" (" suffix=")"/>
+              </names>
+            </if>
+          </choose>
+        </group>
+      </if>
+    </choose>
+  </macro>
+  <macro name="auteurs-ouvrage">
+    <choose>
+      <if type="chapter paper-conference" match="any">
+        <text term="in" suffix=" "/>
+        <group delimiter=", ">
+          <choose>
+            <if variable="author">
+              <names variable="editor" delimiter=", ">
+                <name name-as-sort-order="all" sort-separator=" " and="text" delimiter=", " initialize-with=". " delimiter-precedes-last="never"/>
+                <label form="short" prefix=" "/>
+              </names>
+            </if>
+          </choose>
+          <choose>
+            <if variable="author editor" match="any">
+              <names variable="translator" delimiter=", ">
+                <name name-as-sort-order="all" sort-separator=" " and="text" delimiter=", " initialize-with=". " delimiter-precedes-last="never"/>
+                <label form="short" prefix=" "/>
+              </names>
+            </if>
+          </choose>
+        </group>
+      </if>
+    </choose>
+  </macro>
+  <macro name="titre-ouvrage">
+    <text variable="container-title" form="short" font-style="italic"/>
+    <choose>
+      <if type="article-journal">
+        <text variable="volume" prefix=" " suffix=","/>
+      </if>
+    </choose>
+  </macro>
+  <macro name="lieu-edition">
+    <choose>
+      <if type="book chapter thesis report paper-conference" match="any">
+        <group delimiter=", ">
+          <text variable="publisher-place" form="short"/>
+        </group>
+      </if>
+    </choose>
+  </macro>
+  <macro name="publication">
+    <choose>
+      <if type="thesis">
+        <group prefix=", ">
+          <text value="Thèse de doctorat soutenue à "/>
+          <text macro="lieu-edition"/>
+          <text macro="date-publication" prefix=", "/>
+        </group>
+      </if>
+      <else>
+        <group prefix=", " delimiter=", ">
+          <text macro="lieu-edition"/>
+        </group>
+      </else>
+    </choose>
+  </macro>
+  <macro name="pages-chapitre">
+    <choose>
+      <if type="chapter paper-conference" match="any">
+        <text variable="volume" suffix=" :"/>
```

Comment:
you'll want to set the : as a delimiter rather than as a suffix in case an item doesn't have a page range. Also, I think this should be a non-breaking space before the colon.